third_party/terraform/tests/resource_compute_router_test.go

```diff
@@ -6,177 +6,259 @@ import (
 
 	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
 )
 
-func TestAccComputeRouter_basic(t *testing.T) {
+func TestAccComputeRouterNat_basic(t *testing.T) {
 	t.Parallel()
 
 	testId := acctest.RandString(10)
-	resourceRegion := "europe-west1"
 	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckComputeRouterDestroy,
+		CheckDestroy: testAccCheckComputeRouterNatDestroy,
```

Comment:
It looks like the nat tests somehow ended up in the router test file. Can you get that all sorted out?